src/Persistence/json/jsonconfigstore.py

```diff
@@ -0,0 +1,25 @@
+import json
+from abc import ABC
+
+from Persistence.abstractconfigstore import ConfigStore
+
+
+class JsonConfigStore(ConfigStore, ABC):
+
+    __filename = ""
+
+    def __init__(self, filename='config.json'):
+        self.__filename = filename
+        self.validate()
+
+    def validate(self):
+        with open(self.__filename, 'w+') as file:
+            file.write('{}')
```

Comment:
Won't this overwrite the current file content?

Reply:
Yep. My bad.